src/script/conversation/ConversationCellState.ts

```diff
@@ -348,6 +350,7 @@ const _getStateUnreadMessage = {
         return conversationEntity.isGroup() ? `${messageEntity.unsafeSenderName()}: ${stateText}` : stateText;
       }
     }
+    return '';
```

Comment:
Before it was returning `void` but if it doesn't crash with an empty string then I am also okay with it. :)